skims.cfg

```diff
@@ -1,3 +1,15 @@
+[ztautau_llskim_test]
+    student = llskim.py
+    dataset = mc12_p1344_hadhad_ztautau
+    version = 34
+    testinput = /afs/cern.ch/user/v/vcroft/public/testD3PD
+    dest = ,
+[ztautau_lhskim_test]
+    student = lhskim.py
+    dataset = mc12_p1344_hadhad_ztautau
```

Comment:
@vincecr0ft This line is not OK, you should edit datasets.cfg and put a block for leplep samples and also add a txt file in datasets/leplep/mc.txt with the name of the datasets you want to use